server/channels/app/server.go

```diff
@@ -275,6 +275,8 @@ func NewServer(options ...Option) (*Server, error) {
 	// Step 4: Start platform
 	s.platform.Start()
 
+	s.RegisterBroadcastHooks()
```

Comment:
I'm not happy with having to do this here, but`s.platform.Start()` also starts the websockets and I couldn't find a  way to pass the `broadcastHooks` map into it that felt proper

Reply:
If I do that, I'd also be moving the hooks implementation into the platform package which I was trying to avoid since it feels like more of an app concern. Is that okay?

Reply:
Good point. We can simply pass the hooks map directly from `platform.Start`, and pass them down to `hubStart`.